h/schemas/admin_group.py

```diff
@@ -17,18 +17,39 @@
 _ = i18n.TranslationString
 
 VALID_GROUP_TYPES = (
-    ('private', _('Private')),
     ('restricted', _('Restricted')),
     ('open', _('Open')),
 )
 
 
+def _split_origins(value):
```

Comment:
This feels gross hanging out here but I'm not sure of the best way to integrate it into a schema class.